packages/ag-grid-theme/css/uitk-ag-theme.scss

```diff
@@ -0,0 +1,313 @@
+@import "./uitk-ag-theme-mixin";
+
+.ag-theme-uitk-high-light,
+.ag-theme-uitk-medium-light,
+.ag-theme-uitk-low-light,
+.ag-theme-uitk-touch-light {
```

Comment:
I think this can just be .ag-theme .uitk-light { ... } and repeat for dark? 
I also realised I probably commented to make the tokens --uitkAgGrid, but perhaps --agGrid is better, because we usually give --uitkAgGrid prefix for the CSS API for consumers to override

Reply:
This was the original plan. Ag-grid renders drag-and-drop ghosts outside the grid root (and outside our theme provider actually). It applies the same style to them as was applied to the grid itself. ".uitk-light .uitkAgGrid-..." won't work because there is no .uitk-light on any tag containing dnd ghost.

Reply:
darn. although, if all values are same for each density, do we really need 8 values, or could we just have 2, light and dark: ag-theme-uitk-light/dark ?

Reply:
cc @ararus